### DISCUSSION TITLE
Absence of Code

### DISCUSSION THREAD

**Discussion by AndrewToaster** [TechPenguineer/Hypermc#18] [Q&A] [answered]
Right now I'm confused as to why there is just only a form and 2 (for now) unused classes, when there are about 150 commits. My question is, Is this because of the 'Code Wipe', or am I missing something?

**Comment by FaisalAbusharar** [comment, marked as answer]
there was a wipe, we accidentally deleted important files that we cannot get back, so we needed to reset, there is no record of the old version on github, and also we wanted to reset anyways to make our code more compact and better.

**Reply by AndrewToaster** [reply]
Understandable

**Reply by mstew20** [reply]
So what ended up happening to the Solution file?
